src/serial.rs

```diff
@@ -113,6 +143,13 @@ impl Pins<USART3> for (PD8<Alternate<PushPull>>, PD9<Input<Floating>>) {
     const REMAP: u8 = 0b11;
 }
 
+pub enum WordLength {
+    /// When parity is enabled, a word has 7 data bits + 1 parity bit.
+    _8bits,
+    /// When parity is enabled, a word has 8 data bits + 1 parity bit.
+    _9bits,
```

Comment:
Also I'm not fun of underscores in enums.
Use Bits8, Bits9 as accepted in `stm32-rs`.